google/validation.go

```diff
@@ -107,3 +109,16 @@ func validateRFC1035Name(min, max int) schema.SchemaValidateFunc {
 
 	return validateRegexp(fmt.Sprintf("^"+RFC1035NameTemplate+"$", min-2, max-2))
 }
+
+func validateCloudIoTId(v interface{}, k string) (warnings []string, errors []error) {
```

Comment:
nitpick: "ID" is capitalized when it's short for "identifier"
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done.